### PR TITLE
Cleanup readme, llava examples, usage examples and nccl init

### PR DESCRIPTION
**Pull request by merrymercy**
- Clean up readme
- Clean up llava examples. Deprecate the old llava v1.5/1.6 examples. Use llava-onevision in most examples.
- Separate examples in `examples` folder. Create two subfolder `frontend_language` and `runtime`.
- Fix the bug in #1015
- Other code style improvements

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,12 +22,13 @@ The core features include:
 
 ## News
 - [2024/07] 🔥 Faster Llama3 Serving with SGLang Runtime (vs. TensorRT-LLM, vLLM) ([blog](https://lmsys.org/blog/2024-07-25-sglang-llama3/)).
-- [2024/04] SGLang is used by the official **LLaVA-NeXT (video)** release ([blog](https://llava-vl.github.io/blog/2024-04-30-llava-next-video/)).
+- [2024/08] 🔥 LLaVA-OneVision with single-image, multi-image and video are supported ([blog](https://llava-vl.github.io/blog/2024-08-05-llava-onevision/)).
```
Comment (zhyncs) [inline README.md:25]: May we sort by date?

Comment (merrymercy) [reply README.md:25]: The performance blog is more important so I put it first.

Comment (zhyncs) [reply README.md:25]: make sense

**Comment by zhyncs** [comment]
Since we've already updated the command in the README, we might consider deleting https://github.com/sgl-project/sglang/blob/cleanup/test/srt/test_vision_openai_server.py#L17.

**Review by zhyncs — APPROVED** [review]

File: test/srt/test_vision_openai_server.py [hunk 2]
```diff
@@ -32,11 +28,9 @@ def setUpClass(cls):
             other_args=[
                 "--chat-template",
                 "chatml-llava",
-                "--tokenizer-path",
-                "lmms-lab/llavanext-qwen-siglip-tokenizer",
                 "--chunked-prefill-size",
                 "16384",
-                "--log-requests",
+                # "--log-requests",
```
Comment (zhyncs) [inline test/srt/test_vision_openai_server.py:33]: nit: Maybe we could just delete this.

**Pull request merged**